open_event/views/views.py

```diff
@@ -278,6 +281,38 @@ def generate_icalender_track(track_id):
 	cal.add_component(track)
 	return cal.to_ical()
 
+@app.route('/api/v1/session/<int:session_id>/reviews', methods=['GET'])
+@auto.doc()
+@cross_origin()
+def get_reviews(session_id):
+    """Returns all session's reviews"""
+    reviews = Review.query.filter_by(session_id=session_id)
+    return ObjectFormatter.get_json("reviews", reviews, request)
+
+@app.route('/api/v1/session/<int:session_id>/reviews', methods=['POST'])
+@auto.doc()
+@cross_origin()
+def post_reviews(session_id):
+    """Post reviews for sessions"""
+    data = json.loads(request.data)
+    if "comment" not in data:
+            data["comment"] = ""
+
+    reviews = DataGetter.get_reviews_by_email(data["email"])
+    print reviews.count()
+
+    if reviews.count() > 0:
+        DataManager.update_review(data, reviews[0], session_id)
```

Comment:
Yeah.. I got that wrong.. I have to check for both the user and the session. Will make the change asap.